Dockerfile

```diff
@@ -1,16 +1,15 @@
-# Docker image for the Drone SFTP plugin
-
 FROM alpine:3.3
 
 RUN apk update && \
-    apk add nodejs && \
-    apk add git && \
```

Comment:
Yes, you are right. I missed the currently not executed docker build which is based on that.